README.rst

```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
     cd astroid
     python3 -m pip install -e .
 
+Show your usage
+-----------------
+
+You can place this badge in your README to let others know your project uses pylint.
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
+        :target: https://github.com/PyCQA/pylint
+
+Use the badge in your project's README.md (or any other Markdown file)::
+
+    ![linting: pylint](https://img.shields.io/badge/linting-pylint-green)](https://github.com/PyCQA/pylint)
+
+Use the badge in your projects README.rst (or any other rst file)::
+
+    .. image:: https://img.shields.io/badge/linting-pylint-green
+        :target: https://github.com/PyCQA/pylint
+
 
```

Comment:
No worries if my suggestion is a little _out-of-scope_ 🌨️ . I was just thinking of something like this, which works for folks who have a workflow beginning like:

```yaml
name: pylint
```
```suggestion

If you use GitHub Actions, and one of your CI workflows begins with "name: pylint", you
can use GitHub's
[workflow status badges](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-workflow-file-name)
to show an up-to-date indication of whether pushes to your default branch pass pylint.
```